src/core/client/framework/lib/i18n/format.ts

```diff
@@ -0,0 +1,18 @@
+import { FluentBundle } from "fluent/compat";
+
+export default function format(
+  bundles: FluentBundle[],
+  key: string,
+  args?: object
+): string {
+  const res = bundles.reduce((val, bundle) => {
```

Comment:
Method does not accept a `defaultTo` string. Method does also not warn about missing translations.